helm/cas-data-warehouse/templates/cronjobs/cron-bciers-import.yaml

```
@@ -0,0 +1,122 @@
+{{ $bciers := (lookup "v1" "Secret" (printf "%s-%s" .Values.import.bciers.namespace .Values.environment) .Values.import.bciers.secretName) }}
+
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ template "cas-data-warehouse.fullname" . }}-bciers-import
+  labels:
+{{ include "cas-data-warehouse.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": "pre-upgrade,pre-install"
+    "helm.sh/hook-delete-policy": "before-hook-creation"
+    "helm.sh/hook-weight": "-5"
+spec:
+  suspend: true # This cron job is intended to be triggered manually
+  schedule: "0 * * * *"
+  successfulJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      backoffLimit: 0
+      activeDeadlineSeconds: 600
+      template:
+        metadata:
+          labels:
+{{ include "cas-data-warehouse.labels" . | indent 14 }}
+        spec:
+          activeDeadlineSeconds: 600
+          restartPolicy: Never
+          containers:
+            - env:
+                - name: WAREHOUSE_USER
+                  valueFrom:
+                    secretKeyRef:
+                      key: user
+                      name: {{ template "cas-data-warehouse.fullname" . }}-pg-pguser-datawarehouse
```

```suggestion
                - name: WAREHOUSE_USER
                  valueFrom:
                    secretKeyRef:
                      key: user
                      name: {{ template "cas-data-warehouse.fullname" . }}-pguser-datawarehouse
```

I think this `-pg` can be [left off](https://access.crunchydata.com/documentation/postgres-operator/latest/tutorials/basic-setup/user-management#creating-a-new-user). The other `valueFrom.secretKeyRef.name`s look like they have it as well.

---

`-pg` is added because it's the alias under which we import the `cas-postgres-cluster` chart, the names get concatenated in the `template` function

---

I read through the template to see if I was missing something, and I was right but I just didn't see it 😅.